projects/storefrontapp-e2e-cypress/cypress/integration/regression/site-context/currency/currency-checkout-page.core-e2e-spec.ts

```diff
@@ -9,28 +11,20 @@ describe('Currency switch - checkout page', () => {
   const checkoutReviewPath = siteContextSelector.CHECKOUT_REVIEW_ORDER_PATH;
 
   before(() => {
-    cy.window().then((win) => {
-      win.sessionStorage.clear();
-      win.localStorage.clear();
-    });
+    clearAllStorage();
     cy.requireLoggedIn();
     siteContextSelector.doPlaceOrder();
+    waitForCartPageData(product);
   });
 
   siteContextSelector.stub(
     siteContextSelector.CURRENCY_REQUEST,
     siteContextSelector.CURRENCIES
   );
 
-  describe('populate cart, history, quantity', () => {
```

Comment:
no need to have this as it's not even related to currency. That helper function just plays with the cart quantity to have some kind of cart, but I replaced it with waitForCartPageData(product);. Basically, it adds a product to the cart in the backend for the user/